src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -564,6 +608,13 @@ static int (*next_posix_fallocate64)(int fd, off64_t offset, off64_t len);
 static int (*next_tcgetattr)(int fd, void *termios_p);
 /* end NOT supported by DAOS */
 
+static int (*next_io_setup)(int maxevents, io_context_t *ctxp);
+static int (*next_io_destroy)(io_context_t ctx);
+static int (*next_io_submit)(io_context_t ctx, long nr, struct iocb *ios[]);
+static int (*next_io_cancel)(io_context_t ctx, struct iocb *iocb, struct io_event *evt);
+static int (*next_io_getevents)(io_context_t ctx, long min_nr, long nr, struct io_event *events,
+				struct timespec *timeout);
```

Comment:
it's about time to start creating new files for new features / sets of functions to intercept rather than keep piling on int_dfs.c

Reply:
ok. I am working on putting aio related stuff into separate files. Thank you!